src/Template/Element/advanced_search_form.ctp

```diff
@@ -362,7 +362,7 @@ echo $this->Form->create('AdvancedSearch', [
         </div>
     </div>
 
-    <div class="buttons" layout="<?= $layout ?>" style="align-items:center">
+    <div class="buttons" layout="<?= $layout ?>" style="align-items:<?= (isset($isSidebar) && $isSidebar) ? 'stretch' : 'center'?>">
```

Comment:
You can put this style CSS files instead. There are two files involved:
* `webroot/css/sentences/advanced_search.css`
* `webroot/css/sentences/search.css`

`advanced_search.css` is imported on the advanced search page. It also includes `search.css` with an `@import` CSS directive. The search results page only includes `search.css`.

It looks like you can just set `align-items` to `center` in `advanced_search.css` and leave it unset on the search results page.